thorlcr/thorutil/thormisc.cpp

```diff
@@ -76,6 +76,7 @@ static Owned<IMPtagAllocator> ClusterMPAllocator;
 const StatisticsMapping spillStatistics({StTimeSpillElapsed, StTimeSortElapsed, StNumSpills, StSizeSpillFile, StSizePeakTempDisk});
 const StatisticsMapping soapcallStatistics({StTimeSoapcall});
 const StatisticsMapping basicActivityStatistics({StTimeTotalExecute, StTimeLocalExecute, StTimeBlocked});
+const StatisticsMapping strandedActivityStatistics({StNumThreads}, basicActivityStatistics);
```

Comment:
let's add numThreads to basicActivityStatistics, as many activities use threads, even if project will be the only one for now, that will report it.